packages/aws-rfdk/lib/deadline/test/configure-spot-event-plugin.test.ts

```diff
@@ -432,226 +428,94 @@ describe('ConfigureSpotEventPlugin', () => {
-        spotFleets: [
-          fleetWithCustomProps,
-        ],
-      });
-
-      // THEN
-      cdkExpect(stack).to(haveResourceLike('Custom::RFDK_ConfigureSpotEventPlugin', objectLike({
-        spotFleetRequestConfigurations: objectLike({
-          [groupName]: objectLike({
-            LaunchSpecifications: arrayWith(objectLike({
-              BlockDeviceMappings: arrayWith(objectLike({
-                DeviceName: deviceName,
-                Ebs: objectLike({
-                  SnapshotId: snapshotId,
-                  DeleteOnTermination: deleteOnTermination,
-                  Iops: iops,
-                  VolumeSize: volumeSize,
-                  VolumeType: volumeType,
-                  Encrypted: ABSENT,
-                }),
-                VirtualName: virtualName,
-              })),
-            })),
+            InstanceType: instanceType.toString(),
           }),
-        }),
-      })));
+        }));
+      });
     });
 
-    test('fleet with block devices with no device', () => {
+    test('adds multiple fleet security groups to launch template', () => {
       // GIVEN
-      const deviceName = '/dev/xvda';
-      const volume = BlockDeviceVolume.noDevice();
-
-      const fleetWithCustomProps = new SpotEventPluginFleet(stack, 'SpotEventPluginFleet', {
+      const securityGroups = [
+        new SecurityGroup(stack, 'NewFleetSecurityGroup1', { vpc }),
+        new SecurityGroup(stack, 'NewFleetSecurityGroup2', { vpc }),
+      ];
+      const fleet2 = new SpotEventPluginFleet(stack, 'SpotFleet2', {
         vpc,
         renderQueue,
-        deadlineGroups: [
-          groupName,
-        ],
-        instanceTypes: [
-          InstanceType.of(InstanceClass.T3, InstanceSize.LARGE),
-        ],
+        deadlineGroups: ['group2'],
+        instanceTypes: [new InstanceType('t2.micro')],
         workerMachineImage,
         maxCapacity: 1,
-        blockDevices: [{
-          deviceName: deviceName,
-          volume,
-        }],
+        securityGroups,
       });
 
       // WHEN
       new ConfigureSpotEventPlugin(stack, 'ConfigureSpotEventPlugin', {
         vpc,
-        renderQueue: renderQueue,
-        spotFleets: [
-          fleetWithCustomProps,
-        ],
+        renderQueue,
+        spotFleets: [fleet2],
       });
 
       // THEN
-      cdkExpect(stack).to(haveResourceLike('Custom::RFDK_ConfigureSpotEventPlugin', objectLike({
-        spotFleetRequestConfigurations: objectLike({
-          [groupName]: objectLike({
-            LaunchSpecifications: arrayWith(objectLike({
-              BlockDeviceMappings: arrayWith(objectLike({
-                DeviceName: deviceName,
-                NoDevice: '',
-              })),
-            })),
-          }),
+      cdkExpect(stack).to(haveResourceLike('AWS::EC2::LaunchTemplate', {
+        LaunchTemplateData: objectLike({
+          SecurityGroupIds: fleet2.securityGroups.map(sg => stack.resolve(sg.securityGroupId)),
```

Comment:
nit: can we go with the slightly more explicit test:

```suggestion
          SecurityGroupIds: securityGroups.map(sg => stack.resolve(sg.securityGroupId)),
```

The current code does not necessarily assert that `fleet2.securityGroups` matches what was passed in via the props.

Reply:
Good catch, updated